src/App.jsx

```diff
@@ -14,70 +16,86 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const DEFAULT_GOOD = goods[8];
+  const [titleGood, setTitleGood] = useState(DEFAULT_GOOD);
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState(DEFAULT_GOOD);
```
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/79e0b76b-9815-49bf-9075-188152f357d7)